server/app.ts

```diff
@@ -47,17 +48,17 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpAuthentication())
   app.use(authorisationMiddleware(['ROLE_MANAGE_PRISON_VISITS']))
   app.use(setUpCsrf())
-  app.use(setUpCurrentUser(services))
-  app.use(appInsightsOperationId)
-
   app.get(
     '*',
     dpsComponents.getPageComponents({
       dpsUrl: config.dpsHome,
       logger,
-      // TODO includeMeta: true,
+      includeMeta: true,
     }),
   )
+  app.use(setUpCurrentUser())
+  app.use(populateSelectedEstablishment(services))
```

Comment:
`populateSelectedEstablishment` moved from being within `setupCurrentUser` to included on its own. This means it can be included separately when mocking the app for testing.